### ISSUE TITLE
u-he Presswerk not working

### ISSUE THREAD

**Issue by tas-from-planet-u-he** [free-audio/clap-wrapper#31]
Tested on macOS Monterey 12.6.1 with a 2020 M1 Silicon MacBook Pro using Bitwig 4.4.2.

Audio signals are reaching the wrapped version, at least the input meter reacts accordingly.
No audio is coming out of the plugin, the output meter of Presswerk is stuck at 0dB.
The threshold parameter cannot be moved with the mouse.
The compression graph display is not showing up.

First idea: Maybe the problem is related to Presswerk's side chain feature?

**Comment by defiantnerd** [comment]
Yes, I think it seems to be connected the side chain feature and the audio bus setup.

**Comment by defiantnerd** [comment]
Fixed a faulty bus association and presswerk is now working in Bitwig.

**Comment by tas-from-planet-u-he** [comment]
Did build it again yesterday after cloning the latest clap-wrapper version.
Somehow Presswerk still behaves the same on an M1 Silicon Mac with Monterey.
We'll investigate if there is an issue on our side.

**Comment by Schroedingers-Cat** [comment]
We didn't guard against incoming NaNs which caused some funny things in our UI. This has been fixed.  

However, I'm still seeing Presswerk's parameter `Threshold 1` and `Threshold 2` both being reported as NaN when using the wrapper. I wanted to check whether we're reporting these NaNs under any circumstances to the wrapper but it looks like the wrapper isn't calling paramsValue or paramsValueToText even once. I've also checked multiple places related to these parameters and didn't spot an NaN anywhere within Presswerk.  

ATM, it looks like the wrapper is creating those NaNs. Maybe it's related to the fact that these parameters just have a negative range, but I'm guessing.
